packages/components/rating/src/Rating.stories.tsx

```diff
@@ -9,4 +10,21 @@ const meta: Meta<typeof Rating> = {
 
 export default meta
 
-export const Default: StoryFn = _args => <Rating />
+const sizes: RatingProps['size'][] = ['sm', 'md', 'lg']
+
+export const Default: StoryFn = _args => <Rating defaultValue={3} />
+
+export const Readonly: StoryFn = _args => <Rating defaultValue={3.5} readOnly />
```

Comment:
Maybe for this story we could set "3.6" or "3.62" to show that it rounded to the closest half star value.

Reply:
Some nice a11y definitions:
- A the end of the page: https://component.gallery/components/rating/
- https://baseweb.design/components/rating/

Reply:
@andresin87 Not sure to get your point... Is there any improvement you want to suggest on current a11y definitions here? :thinking: 
@Powerplex I don't have any strong opinion on that, but I wanted to keep thins simple. Also to avoid people thinking we will manage the rounding logic... In our implementation we only "round" to preserve the displayed value... wdyt?